hummingbot/client/command/list_command.py

```diff
@@ -77,10 +77,12 @@ def _get_trades_from_session(self,  # type: HummingbotApplication
                         .filter(TradeFill.timestamp >= start_timestamp)
                         .order_by(TradeFill.timestamp.desc()))
         if number_of_rows is None:
-            result: Optional[TradeFill] = query.all()
+            result: Optional[TradeFill] = query.all() or []
```

Comment:
the type should be `Optional[List[TradeFill]]`.

Reply:
done